content/en/integrations/guide/storage-monitoring-setup.md

```diff
@@ -29,57 +29,63 @@ Select your cloud storage service to access setup instructions.
-
-   Example source-bucket policy:
-
       ```json
-        {
-          "Version": "2012-10-17",
-          "Statement": [
             {
-              "Sid": "AllowListInventoryBucket",
-              "Effect": "Allow",
-              "Action": "s3:ListBucket",
-              "Resource": "arn:aws:s3:::storage-monitoring-s3-inventory-destination"
-            },
-            {
-              "Sid": "AllowGetInventoryObjects",
-              "Effect": "Allow",
-              "Action": "s3:GetObject",
-              "Resource": "arn:aws:s3:::storage-monitoring-s3-inventory-destination/*"
+              "Version": "2012-10-17",
+              "Statement": [
+
+                {
+                  "Sid": "AllowDatadogToEnableInventory",
+                  "Effect": "Allow",
+                  "Action": [
+                    "s3:PutInventoryConfiguration" // Used for onboarding via UI
+                  ],
+                  "Resource": "arn:aws:s3:::storage-monitoring-source-bucket" // source bucket(s)
+                },
+                { //destination bucket
+                  "Sid": "AllowDatadogToReadInventoryFiles",
+                  "Effect": "Allow",
+                  "Action": [
+                    "s3:GetObject",
+                    "s3:ListBucket"
+                  ],
+                  "Resource": [
+                    "arn:aws:s3:::storage-monitoring-inventory-destination",
+                    "arn:aws:s3:::storage-monitoring-inventory-destination/*" // destination bucket(s)/prefix
+                  ]
+                }
+              ]
             }
-          ]
-        }
-
       ```
 
-4. Select the S3 buckets you want to monitor with Storage Monitoring. You can select buckets from multiple AWS accounts at once.
+   2. **Allow source buckets to write to destination buckets.** The destination buckets must include a policy that allows the source buckets to write inventory data. See [Creating a destination bucket policy][502] in the AWS documentation for details.
 
-{{< img src="integrations/guide/storage_monitoring/step-2.png" alt="Select buckets for enabling Storage Monitoring" responsive="true">}}
+   3. Select the S3 buckets you want to monitor with Storage Monitoring. You can select buckets from multiple AWS accounts at once.
 
-5. Assign a destination bucket per region to store S3 inventory reports from the source buckets. This can be an existing AWS bucket or a new one.
+    {{< img src="integrations/guide/storage_monitoring/step-2.png" alt="Select buckets for enabling Storage Monitoring" responsive="true">}}
 
-   - Source bucket: The S3 bucket you want to monitor with Storage Monitoring
-   - Destination bucket: Used to store inventory reports (one per AWS region, can be reused)
+   4. Assign a destination bucket per region to store S3 inventory reports from the source buckets. This can be an existing AWS bucket or a new one.
```

Comment:
```suggestion
   4. Assign one destination bucket per region to store S3 inventory reports from the source buckets. This can be an existing AWS bucket or a new one.
```